source/ftrack_connect_pipeline_qt/plugin/widget/dynamic.py

```diff
@@ -34,6 +36,7 @@ def __init__(
             int: self._build_int_widget,
             float: self._build_float_widget,
             list: self._build_list_widget,
+            DefinitionList: self._build_list_widget,
```

Comment:
IMO is more clear if you check first if its a definitionList, and if it is, convert it to list with definitionList.to_list() method. Otherwise we are leaving the door open to errors passing the DefinitionList to the _build_list_widget method without converting it to list.

Reply:
Agreed, converting it to list.